draft-ietf-mls-protocol.md

```diff
@@ -734,10 +709,31 @@ such an overflow condition occurs.
 
 # Protocol Overview
 
+MLS is designed to operate in the context described in
+{{?I-D.ietf-mls-architecture}}. In particular, we assume that the following
+services are provided:
+
+* An Authentication Service (AS) that enables group members to authenticate the
+  credentials presented by other group members.
+
+* A Delivery Service (DS) that routes MLS messages among the participants in the
+  protocol.
```

Comment:
I would probably keep the sub-bullets describing the usual functions of the DS.

Reply:
```suggestion
* A Delivery Service (DS) that routes MLS messages among the participants in the
  protocol.
  
      * Pre-publication of KeyPackage objects for clients
      * Delivery of Proposal and Commit messages to members of a group
      * Delivery of Welcome messages to new members of a group
      * Sequencing of Commit messages (see {{sequencing}})
```

(Also incorporating some changes from #872)

Reply:
I mean it's up to you, but I actually think this is a regression, because it has a lot of forward references.